src/js/components/homepageUpdate/AwardSearch/AwardSearch.jsx

```diff
@@ -107,11 +103,11 @@ const AwardSearch = () => {
                 const hashData = results.data;
                 if (rankType === "naics" || rankType === "psc") {
                     // eslint-disable-next-line no-unused-expressions
-                    isQAT ? window.open(`/search?hash=${hashData.hash}&section=${section}&type=${rankType}`, "_self") : window.open(`/search?hash=${hashData.hash}&tab=${section}&rankType=${rankType}`, "_self");
```

Comment:
When reviewing this, I noticed the 'jumpTo' behavior was lost once on the advanced search page. I think this has to do with the placeholder component and will need a follow up ticket.